nixos/modules/services/misc/autobrr.nix

```
@@ -0,0 +1,70 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.autobrr;
+in
+{
+  options = {
+    services.autobrr = {
+      enable = mkEnableOption (lib.mdDoc "Autobrr");
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc "Open ports in the firewall for the Autobrr web interface.";
```

```suggestion
        description = "Open ports in the firewall for the Autobrr web interface.";
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557